docs/ref/contrib/postgres/functions.txt

```diff
@@ -28,3 +28,17 @@ Usage example::
     >>> from django.contrib.postgres.functions import TransactionNow
     >>> Article.objects.filter(published__lte=TransactionNow())
     <QuerySet [<Article: How to Django>]>
+
+``RandomUUID``
+==================
+
+.. class:: RandomUUID()
+
+.. versionadded:: 2.0
+
+Returns a version 4 (random) UUID.
```

Comment:
This needs to state that it needs the extension installing.

Also wouldn't hurt to include the canonical example code.

Even better, let's test whether `UUIDField(default=RandomUUID)` works - if it does let's document that both here and in the UUID docs as a note for PG users. (sounds like it doesn't but let's verify)